robot/impl/resource_manager_modular_test.go

```diff
@@ -479,156 +476,6 @@ func (m *dummyModMan) Close(ctx context.Context) error {
 	return nil
 }
 
-func TestDynamicModularComponentLogging(t *testing.T) {
```

Comment:
I will replace this test and the one below with higher level ones (something in the `web/server` package) once I get feedback on my other changes.

Reply:
curious about why we have to move the test

Reply:
This test relies on changes to `LogConfiguration` for a resource actually modifying the logger registry. Modifications to the logger registry now only happen in `web/server/entrypoint.go`. If you're asking why I need to move the test to `web/server`, though, I suspect I'll have to do that to access some private API from `web/server` (particularly `runServer`).